src/app/client/src/app/modules/shared-feature/components/content-player/content-player.component.ts

```diff
@@ -231,9 +232,9 @@ export class ContentPlayerComponent implements OnInit, AfterViewInit, OnDestroy,
   }
 
   copyContent(contentData: ContentData) {
-    const isTypeQuestionset = _.get(contentData,'mimeType') === 'application/vnd.sunbird.questionset';
-    const successMsg = isTypeQuestionset ? this.resourceService.messages.smsg.m0067 :this.resourceService.messages.smsg.m0042;
-    const errorMsg = isTypeQuestionset ? this.resourceService.messages.emsg.m0067 :this.resourceService.messages.emsg.m0008;
+    this.isTypeCopyQuestionset = _.get(contentData,'mimeType') === 'application/vnd.sunbird.questionset';
+    const successMsg = this.isTypeCopyQuestionset ? this.resourceService.messages.smsg.m0067 :this.resourceService.messages.smsg.m0042;
```

Comment:
cant we put this in single if block for both success and error

Reply:
@rajeevsathish can you please review the updated chnages